internal/db/schema/migrations/oss/postgres/42/01_ssh_private_key_passphrase.up.sql

```diff
@@ -0,0 +1,101 @@
+begin;
+
+alter table credential_static_ssh_private_key_credential
+  add column private_key_passphrase_encrypted bytea;
+
+alter table credential_static_ssh_private_key_credential
+  add column private_key_passphrase_hmac bytea;
```

Comment:
* Can these be combined into a single `alter table statement`?
* If there is no password, can we use a sentinel value instead of a null?
* Can we add a constraint check to make sure both values are set or neither value is set?